bench/cmake/optimized/nghttp2_session.c.ll

```diff
@@ -14165,26 +14165,28 @@ define internal fastcc range(i32 -902, 1) i32 @session_pack_extension(ptr nounde
   %13 = load ptr, ptr %12, align 8
   %14 = tail call i64 %11(ptr noundef %0, ptr noundef %6, i64 noundef %spec.select, ptr noundef nonnull %1, ptr noundef %13) #20
   %15 = icmp eq i64 %14, -535
-  br i1 %15, label %24, label %16
+  br i1 %15, label %25, label %16
 
 16:                                               ; preds = %2
-  %17 = icmp ugt i64 %14, %spec.select
-  br i1 %17, label %24, label %18
+  %17 = icmp slt i64 %14, 0
+  %18 = icmp samesign ugt i64 %14, %spec.select
+  %or.cond = select i1 %17, i1 true, i1 %18
```

Comment:
Missing fold

Reply:
Maybe the poison generating flag?
https://godbolt.org/z/Wj49jnW5T